src/c3chart/Area.js

```diff
@@ -8,11 +8,11 @@
 }(this, function (CommonND) {
     function Area(target) {
         CommonND.call(this);
-        this._class = "c3chart_Area";
 
         this._type = "area";
     };
     Area.prototype = Object.create(CommonND.prototype);
+    Area.prototype._class += " c3chart_Area";
```

Comment:
+= is so that Common.js and CommoND.js can now have there own classes added to the div container?